src/tools/sdk/archive/_README.md

```diff
@@ -6,4 +6,4 @@ If you modify that Dart code for this page, don't forget to regenerate
 the JavaScript:
 
     cd src/tools/sdk/archive
-    pub run build_runner build --release --output out
+    dart run build_runner build --release --output out
```

Comment:
This works now? Or should it be `dart pub run build_runner build --release --output out` ?

Reply:
Yes, this works

Reply:
This is gone over in more detail in the (newer) `dart run` page: https://dart.dev/tools/dart-run#in-a-depended-on-package